magic-eight-ball/src/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import "./styles.css";
+
+const API_ENDPOINT = 'https://yesno.wtf/api';
+
+function App() {
+  const [question, setQuestion] = useState("");
+  const [item, setItem] = useState("");
+  const [error, setError] = useState(false);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  function handleOnChange(event) {
+    setQuestion(event.target.value);
+  }
+
+  function submitting() {
```

Comment:
`handleSubmit` would be better name. `submitting` sounds like Boolean variable

Plus, make this function `async` and use `async/await` instead of `Promise.then()`